rollup.config.js

```diff
@@ -19,16 +18,19 @@ const globals = {
 };
 // $FlowFixMe This should be inferred by Flow and manual casting does not work inside of this config.
 const external = Object.keys(globals);
+
 const babelOptions = () => {
   let result = {
     babelrc: false,
-    presets: [['env', { modules: false }], 'react'],
+    presets: [
+      '@babel/preset-env',
+      '@babel/preset-react',
+      '@babel/preset-flow'
+    ],
     plugins: [
       'emotion',
-      'transform-class-properties',
-      'transform-object-rest-spread',
-      'external-helpers',
-    ],
+      '@babel/plugin-proposal-class-properties'
+    ]
```

Comment:
You may remove `babelrc: false` and just add emotion plugin for simplification.

Reply:
good idea, but this an adjacent optimisation related to the .babelrc changes ?)

Reply:
I don't understand what is the problem with this. It's tooling and build scripts upgrade. @gwyneplaine Why can't this be changed here?

Reply:
@TrySound  @DragorWW  This is fine, the main point with the previous .babelrc change comments was to make sure we don't break the lib build via babel-cli, this doesn't have any effect on that part of the build step 👍